app/javascript/components/dashboard-charts-form/areaChart.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { AreaChart } from "@carbon/charts-react";
+
+const AreaChartGraph= ({ data, dataGroups }) => {
+
+  const areaChartOptions = {
```

Comment:
should we use a helper to define all chart options in single file, i feel like maybe some of these can be reused. wdyt?

Reply:
Options are different for each component, but files can be reusable.